### DISCUSSION TITLE
error : Access Denied

### DISCUSSION THREAD

**Discussion by tmoron** [timdorr/tesla-api#553] [General]
hello, I'm trying to get an acess token using nodejs but when I send the request I get this response:
![image](https://user-images.githubusercontent.com/60621486/160253141-5c8b7941-ad49-45b9-b016-f4fb6e504f6b.png)

I don't know if I did something wrong

**Comment by SylvainGa** [comment]
Starting to get this today as well when trying to refresh from a refresh token once the access token had expired

And... it's back. Odd.

**Comment by hilj0637** [comment]
I have the same problem, same the api not work

**Comment by insaneoctane** [comment]
I'm broken too

**Comment by tdorssers** [comment]
You are probably using a WebDriver and if the navigator.webdriver boolean is true then you get this page.
